### ISSUE TITLE
Serving and access to static resources

### ISSUE THREAD

**Issue by oniseijin** [BladeRunnerJS/brjs#1375]
Two issues

1. css map references are not supported in the workbench server
```
GET http://localhost:7070/kendoui-knockout/default/jstree/workbench/v/dev/css/common/bootstrap-theme.css.map 400 (Request did not match {simple-request=css/([^_/:*?"]+)/bundle\.css, language-request=css/([^_/:*?"]+)_([a-z]{2})/bundle\.css, locale-request=css/([^_/:*?"]+)_([a-z]{2})_([A-Z]{2})/bundle\.css})
```
2. No obvious place mentioned in the documents on where to put static resources: json files you may want to serve, or images. Referencing unbundled-resources for example is not clear. br-logo.png is there, but not obvious how to reliably reference it.

**Comment by oniseijin** [comment]
missing seeing this: <img src="<@unbundled-resources@/>/br-logo.png" />

However, how does one access url's via JavaScript?

**Comment by dchambers** [comment]
Yes, you can do the same thing in JavaScript like this:

```js
img.src = require('service!br.app-meta-service').getVersionedBundlePath(
    'unbundled-resources/br-logo.png'));
```

**Comment by andyberry88** [comment]
To add a bit more information to this:
`unbundled-resources` can be used for any file that isn't part of the standard BRJS bundling mechanism, JSON files are a good example of this. Unbundled resources should be kept inside an `unbundled-resources` folder next to `index.html` and referenced either via `<@unbundled-resources@/>/` or `require('service!br.app-meta-service').getVersionedBundlePath(` as above.

To serve images:
if you are referencing them from CSS then they can be placed either inside `resources` or in a theme (discussed on http://bladerunnerjs.org/docs/concepts/assets/) and referenced as a relative path to the CSS file. During the CSS bundling process any `URL('...')` references are automatically rewritten so the images are served by the `css-resource` plugin.

It looks like we've got a gap in the docs for this so I've created #1376 to improve the docs here.
